ci/nightly/pipeline.template.yml

```diff
@@ -283,9 +283,10 @@ steps:
               ]
 
       - id: testdrive-in-cloudtest
-        label: Full Testdrive in Cloudtest (K8s)
+        label: "Full Testdrive in Cloudtest (K8s) %N"
         depends_on: build-aarch64
-        timeout_in_minutes: 300
+        timeout_in_minutes: 180
+        parallelism: 2
```

Comment:
Are you sure this is sufficient? I think `test/cloudtest/test_full_testdrive.py` will need some extension.

Reply:
You were right, done.